vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -121,10 +121,9 @@ The application includes six MFEs in which the above features complement one ano
 
 #### 3. Manage Users
 
-
-
 - The Manage Users MFE makes an API call to Keycloak to fetch user information. When the user is logged into the app, the MFE is visible from the dropdown under the username.
-- By default, application users are not granted Keycloak authorization to manage users. This demonstrates role based access control for an MFE using Keycloak. To enable the Manage Users widget, login to Keycloak and assign the realm-management client's `view-users` and `manage-users` roles to the desired user.
+- By default, application users are not granted Keycloak authorization to manage users. This demonstrates role based access control for an MFE using Keycloak. To enable the Manage Users widget, login to Keycloak and assign the realm-management client's `view-users` and `manage-users` roles to the desired user. Read the
```

Comment:
the repetition of "users" begs differentiation between the two instances to make sense of the sentence. what i read is "application users are not granted keycloak authorization to manage application users," which is a very poorly constructed sentence - circular and redundant. is there another way to interpret this?? i realize you don't want to rewrite unnecessarily but this deserves some tweaking.

log in

is realm-management hyphenated??

Reply:
edited. But it reads exactly the same to me. And unless there is something in error, none of this is objective. We all read things in our own subjective minds, imo

Reply:
yes, we all read things subjectively. but if something is grammatically unclear according to the rules that govern the english language, it is objectively confusing.